packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -508,24 +508,50 @@ async function proxyRequestToAxios(
 	});
 
 	if (configObject.auth?.sendImmediately === false) {
+		// for digest-auth
 		const { auth } = axiosConfig;
 		delete axiosConfig.auth;
-		const header = await axios(axiosConfig).catch((error) => {
+		const resp1 = await axios(axiosConfig).catch((error) => {
```

Comment:
If this requests succeeds (i.e. does not fall into `catch` block, simply return it.